src/main/resources/i18n/Resources_nl.properties

```diff
@@ -66,22 +53,19 @@ supportPaneReportIssueSentence=Problemen? Suggesties?
 supportPaneReportIssueLinkSentence=Creëer ticket hier.
 uploadPaneStopButtonText=Stop
 uploadPaneUploadMoreButtonText=Upload meer...
-#TODO translate
-directoryStructureSupplierProgressTitle=Looking for files
+directoryStructureSupplierProgressTitle=Zoeken naar bestanden
 cloudAlbumsProviderProgressTitle=Het laden van albums in Google Photos
-#TODO translate
-uploaderAlbumProgressTitle=Adding to albums
+uploaderAlbumProgressTitle=Toevoegen aan albums
 uploaderFileProgressTitle=Media bestanden aan het uploaden.
-#TODO translate
-albumManagerProgressStatusTitle=Reconciling albums with Google Photos
+albumManagerProgressStatusTitle=Albums vergelijken met Google Foto's
 # this is the label on top of the failures table; this table is accessible when there are upload failures;
 # each row in the table shows a path to the file and a problem associated with this path
 failuresDialogTopHint=Elke fout hierbeneden geeft 1 item aan. Deze fouten hebben geen invloed op de upload.\
  Wacht tot de upload klaar is. Bekijk de errors, los ze op als het lukt en probeer opnieuw te uploaden. \
  Rijen in deze tabel kunnen worden gekopieerd naar het klembord.
 aboutDialogVersionLabel=Version
 aboutDialogCreditsText=Programming by @ylexus\nDutch translation by Sander
-upgradeDialogTitle=Update Aeschikbaar
+upgradeDialogTitle=Update beschikbaar
```

Comment:
In English, all words in dialog titles and button labels start with capitals - this is the universal convention. Is it not appropriate in Dutch? For example, see button labels just below.

Reply:
You mean that all text in buttons all start with capitals ?
I see it on some places. I will change it to a capital but it must be a B and not a A :)

Reply:
Yes, that’s exactly what I mean. Thanks!